### DISCUSSION TITLE
Naruto Rise of a Ninja slowdown

### DISCUSSION THREAD

**Discussion by acorigins** [xenia-canary/game-patches#140] [General]
Thank you very much for your work. One question, do you know why NRON gives me slowdowns? I'm not talking about FPS drops, it's like the game goes into slow motion at times.
I tried other games like RDR and they work very well, I have no slowdowns, only with naruto.
My pc rx 590 i7 3770k and 12gb ram. I know it's an old pc, I could play at low fps without problems, but the slowdowns do make it annoying.

**Comment by Margen67** [comment]
This is due to the game not using deltatime.
